packages/browser-wallet/src/popup/pages/Account/SendCcd/DisplayToken.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Suspense } from 'react';
+import clsx from 'clsx';
+import Button from '@popup/shared/Button';
+import { TokenMetadata } from '@shared/storage/types';
+import { integerToFractional } from 'wallet-common-helpers';
+import { Atom, useAtomValue } from 'jotai';
+
+interface ChooseTokenProps {
+    balanceAtom: Atom<bigint>;
+    metadata: TokenMetadata;
+    onClick: () => void;
+    disabled?: boolean;
+    className?: string;
+}
+
+interface BalanceProps extends Pick<ChooseTokenProps, 'balanceAtom'> {
+    symbol: string | undefined;
+    display: (amount: bigint) => string | undefined;
+}
+
+function Balance({ balanceAtom, display, symbol }: BalanceProps) {
+    const balance = useAtomValue(balanceAtom);
+
+    return (
+        <p className="m-0">
+            {display(balance) || '0'}
+            {symbol}
+        </p>
+    );
+}
+
+export default function DisplayToken({ metadata, balanceAtom, className, ...props }: ChooseTokenProps) {
```

Comment:
Should we maybe display NFT's in another way? I think it would make more sense to show the name of the NFT instead of the balance, as the icon might be the icon of the NFT marketplace and not the actual image?

Reply:
Well I think we need to display the amount to the user somehow.  Maybe we need @jens-concordium's input for this.

Reply:
![Screenshot from 2022-10-24 15-07-13](https://user-images.githubusercontent.com/87635671/197532705-c564084a-7bc6-4abc-9739-1256bcf28bb5.png)
![Screenshot from 2022-10-24 15-07-07](https://user-images.githubusercontent.com/87635671/197532709-f8ad6217-d9b4-42ac-81ed-e242bafd4275.png)

For reference.

Reply:
I've tried to display the name